src/styles/blocks/header.scss

```diff
@@ -0,0 +1,14 @@
+.header {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  height: 60px;
+  padding-right: 50px;
+
+  &-left {
+    width: 40px;
+    height: 40px;
+    padding-top: 10px;
+    padding-left: 50px;
+  }
```

Comment:
The class `header-left` should be a separate block or element, not a modifier of the `header` block. Consider renaming it to something like `header__left` to follow BEM conventions more accurately.